lib/livebook_web/live/session_live/shortcuts_component.ex

```diff
@@ -143,6 +143,12 @@ defmodule LivebookWeb.SessionLive.ShortcutsComponent do
         press_all: true,
         desc: "Save notebook",
         basic: true
+      },
+      %{
+        seq: ["ctrl", "alt", "-"],
+        seq_mac: ["⌃", "⌥", "-"],
+        press_all: true,
+        desc: "Go back to last editor"
```

Comment:
```suggestion
        desc: "Go back to previous editor"
```

Reply:
Let's implement go forward as part of this PR too!

Reply:
Which keyboard shortcut we should use for `go forward`?

Reply:
If we're using `ctrl` + `⌥` + `-` (`ctrl` + `alt` + `-`) for goin back, maybe we can use `ctrl` + `⌥` + `+` (`ctrl` + `alt` + `+`) for going forward ?

Reply:
wouldn't we need to use `shift` to type the key `+`?

Reply:
> wouldn't we need to use shift to type the key +?

oops, that's right🤦

Reply:
`ctrl`+`alt`+`shift`+`-` is probably too much, so maybe `cltr`+`alt`+`=`?